apps/champions/priv/skills.json

```diff
@@ -20,35 +1663,339 @@
+      }
+    ]
+  },
+  {
+    "name": "H4ck Basic",
+    "energy_regen": 100,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+          "type": "instant"
+        },
+        "initial_delay": 0,
+        "components": [],
+        "modifier": [],
+        "executions": [
+          {
+            "type": "DealDamage",
+            "attack_ratio": 1,
+            "energy_recharge": 50,
+            "delay": 0
+          }
+        ],
+        "target_strategy": "backline",
+        "target_count": 1,
+        "target_allies": false
+      }
+    ],
+    "cooldown": 5
+  },
+  {
+    "name": "H4ck Ultimate",
+    "energy_regen": 0,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+          "duration": 2,
+          "period": 1
+        },
+        "initial_delay": 0,
+        "components": [],
+        "modifiers": [
+          {
+            "attribute": "speed",
+            "modifier_operation": "Multiply",
+            "magnitude_calc_type": "Float",
+            "float_magnitude": 0
+          }
+        ],
+        "executions": [
+          {
+            "type": "DealDamage",
+            "attack_ratio": 0.49,
+            "energy_recharge": 50,
+            "delay": 0
+          }
+        ],
+        "target_strategy": "all",
+        "target_count": 6,
+        "target_allies": false
+      }
+    ]
+  },
+  {
+    "name": "Kenzu Basic",
+    "energy_regen": 100,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+          "type": "instant"
+        },
+        "initial_delay": 0,
+        "components": [],
+        "modifier": [],
+        "executions": [
+          {
+            "type": "DealDamage",
+            "attack_ratio": 0.8,
+            "energy_recharge": 50,
+            "delay": 0
+          },
+          {
+            "type": "DealDamage",
+            "attack_ratio": 0.8,
+            "energy_recharge": 50,
+            "delay": 0
+          }
+        ],
+        "target_strategy": "nearest",
+        "target_count": 1,
+        "target_allies": false
+      }
+    ],
+    "cooldown": 5
+  },
+  {
+    "name": "Kenzu Ultimate",
+    "energy_regen": 0,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+          "type": "instant"
+        },
+        "initial_delay": 0,
+        "components": [],
+        "modifiers": [],
+        "executions": [
+          {
+            "type": "DealDamage",
+            "attack_ratio": 0.38,
+            "energy_recharge": 50,
+            "delay": 0
+          },
+          {
+            "type": "DealDamage",
+            "attack_ratio": 0.59,
+            "energy_recharge": 50,
+            "delay": 0
+          },
+          {
+            "type": "DealDamage",
+            "attack_ratio": 0.83,
+            "energy_recharge": 50,
```

Comment:
We can ask for the correct value later, but we should set these delays to increasing numbers like 0, 1, 2 to correctly emulate the spirit of the skill